dockers/docker-database/critical_processes.j2

```diff
@@ -1,5 +1,6 @@
 {% if INSTANCES %}
 {%     for redis_inst, redis_items in INSTANCES.items() %}
+
```

Comment:
Remove this unnecessary change?

Reply:
If there is no blank line here, all program:{{ redis_inst }} of critical_processes will be concentrated on one line, which will cause subsequent regular expression matching to fail.

Reply:
#Closed

Reply:
Does changing `{%- endfor %}` to `{% endfor %}` work? That will make the intention a bit clearer.

Reply:
Changing {%- endfor %} to {% endfor %} works. I apply this change instead of adding blank line.